.eslintrc.js

```diff
@@ -0,0 +1,234 @@
+		'arrow-body-style'        : ['error', 'as-needed', {
+			requireReturnForObjectLiteral : true,
+		}],
+		'arrow-parens'            : ['error', 'as-needed'],
+		'arrow-spacing'           : 'error',
+		'constructor-super'       : 'error',
+		'no-class-assign'         : 'error',
+		'no-confusing-arrow'      : 'error',
+		'no-const-assign'         : 'error',
+		'no-dupe-class-members'   : 'error',
+		'no-duplicate-imports'    : 'error',
+		'no-new-symbol'           : 'error',
+		'no-this-before-super'    : 'error',
+		'no-useless-computed-key' : 'error',
+		'no-useless-constructor'  : 'error',
+		'no-useless-rename'       : 'error',
+		'no-var'                  : 'error',
+		'object-shorthand'        : 'error',
+		'prefer-arrow-callback'   : 'error',
+		'prefer-const'            : 'error',
+		'prefer-rest-params'      : 'error',
+		'prefer-spread'           : 'error',
+		'require-yield'           : 'error',
+		'rest-spread-spacing'     : 'error',
+		'template-curly-spacing'  : ['error', 'always'],
+		// Miscellaneous
+		'strict' : ['error', 'never'],
+		// React
+		'react/no-danger'                 : 'error',
+		'react/no-deprecated'             : 'error',
+		'react/no-did-mount-set-state'    : 'error',
+		'react/no-did-update-set-state'   : 'error',
+		'react/no-direct-mutation-state'  : 'error',
+		'react/no-is-mounted'             : 'error',
+		'react/no-render-return-value'    : 'error',
+		'react/no-unknown-property'       : 'error',
+		'react/prefer-es6-class'          : 'error',
+		'react/prefer-stateless-function' : 'warn',
+		//'react/prop-types'                : 'error',
+		'react/react-in-jsx-scope'        : 'error',
+		'react/require-render-return'     : 'error',
+		'react/self-closing-comp'         : ['error', { component : true, html : true }],
+		// JSX
+		'react/jsx-boolean-value'            : 'error',
+		'react/jsx-closing-bracket-location' : ['error', 'line-aligned'],
+		'react/jsx-curly-spacing'            : ['error', 'always'],
+		'react/jsx-equals-spacing'           : 'error',
+		//'react/jsx-filename-extension'       : 'error',
+		'react/jsx-first-prop-new-line'      : ['error', 'multiline'],
+		'react/jsx-indent'                   : ['error', 'tab'],
+		'react/jsx-indent-props'             : ['error', 'tab'],
+		'react/jsx-key'                      : 'error',
+		'react/jsx-no-bind'                  : 'error',
+		'react/jsx-no-comment-textnodes'     : 'error',
+		'react/jsx-no-duplicate-props'       : 'error',
+		'react/jsx-no-target-blank'          : 'error',
+		'react/jsx-no-undef'                 : 'error',
+		'react/jsx-pascal-case'              : 'error',
+		'react/jsx-space-before-closing'     : 'error',
+		'react/jsx-uses-react'               : 'error',
+		'react/jsx-uses-vars'                : 'error',
+		'react/jsx-wrap-multilines'          : 'error',
+	},
+	plugins : [
+		'react',
+	],
+	settings : {
+		react : {
+			version : require('./package.json').dependencies.react,
+		},
+	},
+	globals: {
+		expect: true
```

Comment:
Just checked and it looks that CRA uses `Jest` by default. I'm not really familiar with it, but it looks like theses are globals. https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#writing-tests